.github/scripts/generate_ci_workflows.py

```diff
@@ -156,11 +137,10 @@ def assert_valid(self) -> None:
             assert self.test_runner_type in WINDOWS_RUNNERS, err_message
 
     def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
-        output_file_path = GITHUB_DIR / f"workflows/generated-{workflow.build_environment}.yml"
+        output_file_path = GITHUB_DIR / f"workflows/generated-{self.build_environment}.yml"
         with open(output_file_path, "w") as output_file:
-            GENERATED = "generated"
-            output_file.writelines([f"# @{GENERATED} DO NOT EDIT MANUALLY\n"])
-            output_file.write(workflow_template.render(asdict(workflow)))
+            output_file.writelines(["# @generated DO NOT EDIT MANUALLY\n"])
```

Comment:
The reason why this was a variable before was because if `@generated` is anywhere in a file then phabricator picks it up as a generated change, can we please move it back to a variable so that we can still see this file on phab?

Reply:
oh, I definitely will add a comment for this, good to know